NebulaHost/MultiplayerHostSession.cs

```diff
@@ -14,6 +14,7 @@ namespace NebulaHost
     public class MultiplayerHostSession : MonoBehaviour, INetworkProvider
     {
         public static MultiplayerHostSession Instance { get; protected set; }
+        public static bool isDedicated = false;
```

Comment:
You should put this flag in the `LocalPlayer` class that we you will be able to do checks like `LocalPlayer.isDedicated` instead of `LocalPlayer.Data == null`